functions.Rmd

```diff
@@ -14,6 +14,24 @@ row_adder()
 permutation()
 ```
 
+## Anonymous functions
+
+If using an anonymous function, prefer the new lambda syntax: `\(x) {}`.
+
+```R
+# Good
+map(xs, \(x) mean((x + 5)^2))
+
+# Bad
+map(xs, function(x) mean((x + 5)^2))
+map(xs, ~ mean((.x + 5)^2))
+```
+
+* Use informative argument names.
```

Comment:
This rule is not specific to anonymous functions; maybe it belongs in the `## Naming` section?

```suggestion
* As with any function, prefer informative argument names.
```

(or,